src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_CAPACITY = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private int capacity;
+
```

Comment:
I think it's redundant to create variable for capacity, you can always use table.length instead